configuration/external_parameters.rst

```diff
@@ -285,7 +285,8 @@ Symfony provides the following env var processors:
 
             # config/packages/security.yaml
             parameters:
-                env(HEALTH_CHECK_METHOD): 'Symfony\Component\HttpFoundation\Request::METHOD_HEAD'
+                env(HEALTH_CHECK_METHOD): !php/const Symfony\Component\HttpFoundation\Request::METHOD_HEAD
```

Comment:
Did this just not work before? I'm confused by how we use `!php/const` here, but also the `const:` env processor below. This section is supposed to be talking about that `const:` processor. I don't understand these changes

Reply:
Yeah, it's mixing the PHP constant tag from the Yaml component with the `const:` processor of the DI component. The const processor expects the name of the constant and will fail otherwise:

https://github.com/symfony/dependency-injection/blob/35cafad671aaf85da229e6139f493763a66f04fd/EnvVarProcessor.php#L155-L161

So we should either revert this change or remove the `const:` processor

Reply:
My bad, I did not see it was about `const:` env processor. Reverted, thanks!